plugins/apitracing/src/lib/FunctionHook.cpp

```diff
@@ -46,9 +46,12 @@ namespace ApiTracing
     {
         if (std::find(hookedProcesses.begin(), hookedProcesses.end(), event.getCr3()) != hookedProcesses.end())
         {
-            logger->info(
-                "hookCallback hit",
-                {{"Module", moduleName}, {"Function", functionName}, {"Gla", fmt::format("{:x}", event.getGla())}});
+            logger->info("hookCallback hit",
```

Comment:
I would consider this debug level.

Reply:
There is a lot of information gathered here. I would be careful with this, considering this function will be called rather frequently. Imho it would make sense to guard this and other similar logs with an `#ifdef`